datafusion/physical-expr-common/Cargo.toml

```diff
@@ -37,6 +37,7 @@ path = "src/lib.rs"
 
 [dependencies]
 arrow = { workspace = true }
+arrow-array = { workspace = true }
```

Comment:
could we use `use arrow::datatypes::Float64Type;` to avoid re-import? `arrow-array` is sub-crate in `arrow`

Reply:
Thanks @jayzhan211. It looks better.